mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1424,7 +1428,17 @@ def _log_eval_table(self):
             justifications = metric_value.justifications
 
             if scores:
-                columns[f"{metric_name}/score"] = scores
+                append_score = True
+                if metric_name.startswith(metric_prefix) and metric_name[len(metric_prefix) :] in [
+                    _TOKEN_COUNT_METRIC_NAME,
+                    _LATENCY_METRIC_NAME,
+                ]:
+                    append_score = False
+
+                if append_score:
+                    columns[f"{metric_name}/score"] = scores
+                else:
+                    columns[metric_name] = scores
```

Comment:
small suggestion to avoid using the `appendScore` temp var

```suggestion
                if metric_name.startswith(metric_prefix) and metric_name[len(metric_prefix) :] in [
                    _TOKEN_COUNT_METRIC_NAME,
                    _LATENCY_METRIC_NAME,
                ]:
                    columns[metric_name] = scores
                else:
                    columns[f"{metric_name}/score"] = scores
```